configuration/concepts/drugs_concepts.csv

```diff
@@ -311,4 +312,109 @@ b2727136-70f5-470b-81a2-26aeff3f346f,,,"Levonorgestrel, Ethinylestradiol","Levon
+468716a5-6e90-42c1-b870-59ae0911875a,,,Insulin NPH,Insulin NPH,Drug,N/A,,,
+6e1f9544-89f0-4fb4-a0b1-fa4d457da618,,,Insulin 70/30,Insulin 70/30,Drug,N/A,,,
+ce130684-6cc5-4495-8c9e-24997c4d897f,,,Albumin,Albumin,Drug,N/A,,,
+2dbd29cf-a0bc-4bfc-bb20-95dfd8c05245,,,Dextran,Dextran,Drug,N/A,,,
+02a15b77-9093-4265-b422-4c4b218fc489,,,Gelofusine,Gelofusine,Drug,N/A,,,
+409beedf-4f72-4f1c-9478-908917c11a8f,,,Calcium chloride,Calcium chloride,Drug,N/A,,,
+800bb234-aabc-449c-ad8e-7510446dc1b1,,,Sodium bicarbonate,Sodium bicarbonate,Drug,N/A,,,
+861a302b-cb47-4fc4-82ac-65ba4e971ff9,,,Aluminum & Magnesium hydroxide & Simethicone,Aluminum & Magnesium hydroxide & Simethicone,Drug,N/A,,,
+93c9e6d7-71f0-4817-8e64-1c12469cff4c,,,Bisacodyl,Bisacodyl,Drug,N/A,,,
+ecff0e05-2d5d-40b8-8112-22ee733b17a1,,,Sodium phosphate (Fleet's),Sodium phosphate (Fleet's),Drug,N/A,,,
+2f427b27-c1ba-43e8-a3e1-bda24d9ec103,,,"Nitrofurasone, Neomycin, Dexamethasone (Otofuran®)","Nitrofurasone, Neomycin, Dexamethasone (Otofuran®)",Drug,N/A,,,
+c2d40d92-735f-42fe-88e2-be50a53c8f97,,,Latanoprost,Latanoprost,Drug,N/A,,,
+6f53a9b8-b484-4899-a020-033500546631,,,Trobramycin,Trobramycin,Drug,N/A,,,
+12fb801c-7fde-48d1-b985-b14ce268c80c,,,Tobramycin & Dexa,Tobramycin & Dexa,Drug,N/A,,,
+cecfff9f-6e73-4c65-b4a2-43883973b0c5,,,Ketorolac,Ketorolac,Drug,N/A,,,
+4bfe538d-32d9-40bc-84de-ff5dc043a301,,,Hialorunate de Na&,Hialorunate de Na&,Drug,N/A,,,
+56b34ea9-95f6-42ea-a582-fc513494602e,,,Tetracycline,Tetracycline,Drug,N/A,,,
+23df848e-435a-4f8a-b6bc-27235b98e018,,,Timolol,Timolol,Drug,N/A,,,
+d37b5fdb-50c3-40f2-906d-7c81bc22649b,,,Predisolone,Predisolone,Drug,N/A,,,
+e5a0d185-4c6a-42c8-a21e-e7436720d03c,,,Sodium hyaluronate & Oxymetazoline,Sodium hyaluronate & Oxymetazoline,Drug,N/A,,,
+8ebe6adf-72e3-495e-860f-c4a71ffc8016,,,Sodium chloride,Sodium chloride,Drug,N/A,,,
+3ea490ec-60e1-446e-83ca-6f5d1da8c2e6,,,Brimonidine tartrate,Brimonidine tartrate,Drug,N/A,,,
+b39042a0-345a-478c-8f8c-91286aa066d6,,,Betaxolol,Betaxolol,Drug,N/A,,,
+04c434fc-ab58-4a4f-84c5-18e6eaa08646,,,Brimonidine & timolol,Brimonidine & timolol,Drug,N/A,,,,
+3d0fed4d-1fd4-44a4-99fd-d59401067134,,,Dorzolamine & timolol,Dorzolamine & timolol,Drug,N/A,,,
+975c29db-ab3a-4c08-a3fb-829f79a6a7e8,,,Pilocarpine,Pilocarpine,Drug,N/A,,,
+32ed8cb1-bc73-44aa-934d-bf7626154af8,,,Cyclopentolate,Cyclopentolate,Drug,N/A,,,
+c5e3d595-76ae-4bf0-81d2-b3db54c36e6b,,,Propacaine,Propacaine,Drug,N/A,,,
+8d6744e1-4607-4cef-abd6-0827113e9cbe,,,Phenylephrine,Phenylephrine,Drug,N/A,,,
+e1b3ec78-175f-449b-a689-250641da6a05,,,Salbutamol & Iprotopium bromide,Salbutamol & Iprotopium bromide,Drug,N/A,,,
+b2841832-add7-4c6d-aa6e-425b97bb717d,,,Alcohol,Alcohol,Drug,N/A,,,
+66010c7d-dd21-4f83-b950-e2fff1ff6a73,,,Chlorhexidine,Chlorhexidine,Drug,N/A,,,
+cee47202-131e-46b9-8d23-3321cc03f750,,,Eucerin,Eucerin,Drug,N/A,,,
+f3ea42fb-f51f-4083-8b63-d787f8e991c5,,,Hemarrhoidal,Hemarrhoidal,Drug,N/A,,,

```

Comment:
Double check if this one is used anywhere, otherwise, remove it.

Reply:
Changed it to a generic name. Used in HSC